frontend/src/app/main/data/workspace/selection.cljs

```diff
@@ -565,7 +565,9 @@
 (defn calc-duplicate-delta
   [obj state objects]
   (let [{:keys [id-original id-duplicated]}
-        (get-in state [:workspace-local :duplicated])]
+        (get-in state [:workspace-local :duplicated])
+        move? (and (cph/frame-shape? obj)
+                   (not (:component-root? obj)))]
```

Comment:
Perhaps what you want here is `(not (ctk/component-head obj))`

Reply:
Almost! It's `(not (ctk/instance-head? obj))`